src/battle_animation.h

```diff
@@ -62,6 +62,27 @@ class BattleAnimation : public Sprite {
 	/** @return true if the animation only plays audio and doesn't display **/
 	bool IsOnlySound() const;
 
+	/**
+	 * Set if the animation is inverted
+	 *
+	 * @param inverted if the animation is inverted
+	 **/
+	void SetInvert(bool inverted);
+
+	/**
+	 * Set if the animation ignores vertical offset
+	 *
+	 * @param nooffset if the animation ignores vertical offset
+	 **/
+	void SetNoOffset(bool nooffset);
```

Comment:
What is this vertical offset? Can you tell me how to reproduce this?
Also a negation in a Set is confusing: ``SetOffset`` or because this only affects the vertical offset: ``SetIgnoreVerticalOffset`` and vertical is Y so it could also be ``SetIgnoreYOffset``

Reply:
This is difficult to explain but this vertical offset check is needed if the target is a battle animation sprite. In RPG_RT the position setting (top, center of bottom) of a battle animation is ignored if the target is a battle animation sprite. You can test this in EasyRPG (current master) in the game "Vampires Dawn II" fighting against the "Horanmagier (battle ID 62)". When he casts "Eisiger Entzug" you will notice that the animation is shown too far above.